lib/iris/fileformats/netcdf/saver.py

```diff
@@ -1864,8 +1864,20 @@ def _create_cf_grid_mapping(self, cube, cf_var_cube):
         None
 
         """
-        cs = cube.coord_system("CoordSystem")
-        if cs is not None:
+        # TODO(CB): We need ALL coord systems here, not just the fist one found.
+        # cs = cube.coord_system("CoordSystem")
+
+        # TODO(CB): This is workaround code (gets _unique_ coord systems):
+        # TODO(CB): Put this code in new cube.coord_systems() function, or modify cube.coord_system()
+        # coord_systems = [coord.coord_system for coord in cube.coords() if coord.coord_system]
```

Comment:
This whole area is a mess IMO; see #6030

Reply:
Resolved - see https://github.com/SciTools/iris/issues/6030#issuecomment-3036693387